addons/account/tests/test_account_payment.py

```diff
@@ -512,3 +512,70 @@ def test_payments_copy_data(self):
 
         self.assertEqual(duplicate_payment_1.amount, payment_1.amount)
         self.assertEqual(duplicate_payment_2.amount, payment_2.amount)
+
+    def test_payments_invoice_payment_state_without_outstanding_accounts(self):
+        """ Ensures that, without an outstanding accounts set on the bank journal payment method,
+            the invoice payment state is correctly computed, as well as the amount residual """
+        def register_payment(move, amount):
+            return self.env['account.payment.register'].with_context(active_model='account.move', active_ids=move.ids).create({
+                'amount': amount,
+            })._create_payments()
+
+        def create_statement_line_and_reconcile(move_to_reconcile_with, amount):
```

Comment:
"Community edition", and reconcile with a bank statement? :thinking: 
Isn't that impossible?